ui/src/components/NodePagePodsTab.spec.tsx

```diff
@@ -0,0 +1,124 @@
+import { brand } from '@scality/core-ui/dist/style/theme';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { useSelector } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
+import { mockOffsetSize } from '../tests/mocks/util';
+import translations_en from '../translations/en.json';
+import NodePagePodsTab from './NodePagePodsTab';
+
+const mockUseSelector = useSelector as jest.Mock;
+
+const mockResult = (numContainerRunning: number, status = 'Running') => ({
+  name: 'test-pod',
+  age: '4d2h',
+  namespace: 'xcore',
+  status: {
+    status,
+    numContainer: 3,
+    numContainerRunning,
+  },
+  log: 'test',
+});
```

Comment:
OK. because of the implement of the component is not able to do unit test. We have to render to the entire Node page in order to use msw to mock the api response.